### PR TITLE
libvirt: simplify how bootstrap_dns is passed to terraform

### PR DESCRIPTION
**Pull request by jstuever**
This change simplifies libvirt's passing of the bootstrap_dns variable to
terraform when tearing down the bootstrap components. It is now more like like
the corresponding gcp code. Instead of writing a vars file to disk, it is now
set as a command line argument.

### CODE REVIEW

File: pkg/destroy/bootstrap/bootstrap.go [hunk 1]
```diff
@@ -77,7 +66,7 @@ func Destroy(dir string) (err error) {
 			return errors.Wrap(err, "Terraform apply")
 		}
 	case libvirt.Name:
-		_, err = terraform.Apply(tempDir, platform, extraArgs...)
+		_, err = terraform.Apply(tempDir, platform, append(extraArgs, "-var=bootstrap_dns=false")...)
```
Comment (markmc) [inline pkg/destroy/bootstrap/bootstrap.go]: A comment above this line would be nice, like:

```
// First remove the bootstrap node from DNS
```

File: pkg/destroy/bootstrap/bootstrap.go [hunk 2]
```diff
@@ -30,17 +30,6 @@ func Destroy(dir string) (err error) {
 	tfPlatformVarsFileName := fmt.Sprintf(cluster.TfPlatformVarsFileName, platform)
 	copyNames := []string{terraform.StateFileName, cluster.TfVarsFileName, tfPlatformVarsFileName}
```
Comment (markmc) [inline pkg/destroy/bootstrap/bootstrap.go]: Let's revert the addition of the copyNames variable that commit 5e285cc added - we don't need it anymore

Comment (zeenix) [reply pkg/destroy/bootstrap/bootstrap.go]: Looks good to me too, apart from what @markmc pointed out.

File: pkg/destroy/bootstrap/bootstrap.go (hunk 1 above)
Comment (jstuever) [reply pkg/destroy/bootstrap/bootstrap.go]: Done.

File: pkg/destroy/bootstrap/bootstrap.go (hunk 2 above)
Comment (jstuever) [reply pkg/destroy/bootstrap/bootstrap.go]: Great catch... done.

**Comment by zeenix** [comment]
On bot's instructions:
/assign @crawford

**Pull request merged**